src/client/Microsoft.Identity.Client/Internal/ClientCredential/IClientCredential.cs

```diff
@@ -11,12 +11,15 @@
 using Microsoft.Identity.Client.Core;
 using Microsoft.Identity.Client.OAuth2;
 using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+using Microsoft.Identity.Client.TelemetryCore;
 using Microsoft.Identity.Client.Utils;
 
 namespace Microsoft.Identity.Client.Internal.ClientCredential
 {
     internal interface IClientCredential
     {
+        AssertionType TelemetryDetails { get; }
```

Comment:
`LogAssertionType` or `AssertionTypeForTelemetry` instead of `TelemetryDetails`?